python/tk_multi_workfiles/my_tasks/my_tasks_form.py

```diff
@@ -1,395 +1,63 @@
-            self._ui.new_task_btn.setEnabled(False)
-        else:
-            self._ui.new_task_btn.hide()
+        # There is no need for the my tasks toggle.
+        self._ui.my_tasks_cb.hide()
 
+        # Sets an item delete to show a list of tiles for tasks instead of nodes in a tree.
         self._item_delegate = None
-        if tasks_model:
-
-            if True:
-                # create a filter proxy model between the source model and the task tree view:
-                filter_fields = ["content", {"entity":"name"}]
-                filter_fields.extend(tasks_model.extra_display_fields)
-                filter_model = EntityProxyModel(self, filter_fields)
-                monitor_qobject_lifetime(filter_model, "My Tasks filter model")
-                filter_model.rowsInserted.connect(self._on_filter_model_rows_inserted)
-                filter_model.setSourceModel(tasks_model)
-                self._ui.task_tree.setModel(filter_model)
-
-                # create the item delegate - make sure we keep a reference to the delegate otherwise 
-                # things may crash later on!
-                self._item_delegate = MyTaskItemDelegate(tasks_model.extra_display_fields, self._ui.task_tree)
-                monitor_qobject_lifetime(self._item_delegate)
-                self._ui.task_tree.setItemDelegate(self._item_delegate)
-
-                self._ui.search_ctrl.search_edited.connect(self._on_search_changed)
-            else:
-                self._ui.task_tree.setModel(tasks_model)
-
-        # connect to the selection model for the tree view:
-        selection_model = self._ui.task_tree.selectionModel()
-        if selection_model:
-            selection_model.selectionChanged.connect(self._on_selection_changed)
+        if tasks_model and True:
+            # create the item delegate - make sure we keep a reference to the delegate otherwise
+            # things may crash later on!
+            self._item_delegate = MyTaskItemDelegate(tasks_model.extra_display_fields, self._ui.entity_tree)
+            monitor_qobject_lifetime(self._item_delegate)
+            self._ui.entity_tree.setItemDelegate(self._item_delegate)
 
     def shut_down(self):
         """
         Clean up as much as we can to help the gc once the widget is finished with.
         """
         signals_blocked = self.blockSignals(True)
         try:
-            # clear any references:
-            self._current_item_ref = None
-
-            # clear the selection:
-            if self._ui.task_tree.selectionModel():
-                self._ui.task_tree.selectionModel().clear()
-
-            # detach the filter model from the view:
-            view_model = self._ui.task_tree.model()
-            if view_model:
-                self._ui.task_tree.setModel(None)
-                if isinstance(view_model, EntityProxyModel):
-                    view_model.setSourceModel(None)
-
+            EntityTreeForm.shut_down(self)
```

Comment:
I'm torn about it too. But in workfiles 2 case, it was originally written not using super so I've followed the pattern.